release-notes.md

```diff
@@ -1,3 +1,36 @@
+# Kajam-11.13.0 #
+
+## Mediaprep Tvinci distribution profile use dynamic tags ##
+
+ - Issue Type: Story
+ - Issue ID: PLAT-4617
+
+#### Configuration ####
+- None.
+
+#### Deployment scripts####
+
+	php /opt/kaltura/app/deployment/updates/scripts/2016_04_05_migrate_tvinci_distribution_tags_to_be_dynamic.php <partner_id>
```

Comment:
Since the script requires <partner id> I think you should mention which partners to run it on